.github/workflows/update-images.yaml

```
@@ -0,0 +1,55 @@
+name: Update Cloud Images
+
+on:
+  workflow_dispatch:
+  schedule:
+  # 04:11 UTC every Sunday
+  - cron: 11 4 * * 0
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  run-scraper:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.13'
```

Might as well use the latest Python here... (and the fact that it's Python version π is fun :))
```suggestion
          python-version: '3.14'
```